### ISSUE TITLE
Refactor deploy notification

### ISSUE THREAD

**Issue by AronNovak** [Gizra/drupal-starter#483]
The DDEV ENV variable passing is not reliable, it breaks for more complex commit messages.
We might pass the commit message via a `gitignore`-ed text file to avoid deployment instability due to commit message - YAML syntax issues.

**Comment by amitaibu** [comment]
Can you give an example of a problematic commit message?

**Comment by AronNovak** [comment]
```
Global configuration:

instrumentation-opt-in=false

omit-containers=[dba]

web-environment=[ remove not:prose as it is broken (#214)

 Add header text styles, remove prose as it is broken

 More adjustments. Styleguide helper

 Fix coding standards,TERMINUS_MACHINE_TOKEN=[secure],TRAVIS_COMMIT_MESSAGE=Add header text styles]

mutagen-enabled=false
```

An example Travis output.

**Comment by AronNovak** [comment]
https://github.com/ddev/ddev/issues/4791

**Comment by GizraNotifier** [comment]
The latest merged PR #493 just got deployed successfully to Pantheon [`qa`](https://gizra-drupal-starter-qa.pantheonsite.io) environment
